docs/madr/decisions/008-mesh-logging.md

```diff
@@ -0,0 +1,381 @@
+            address: 127.0.0.1:5000
+```
+
+##### Positive Consequences
+
+* consistency in regards to `enabledBackend`
+* easier to have per zone backends
+
+##### Negative Consequences
+
+* less dry
+
+### Other configuration options
+
+#### Format
+
+Format is currently defined on a `backend`, but in reality it's not really tied to a backend 
+(multiple backends can have the same format).
+
+##### Leave `format` in the `backend`
+
+###### Positive Consequences
+
+* users are familiar with this
+
+###### Negative Consequences
+
+* less dry
+
+##### Move `format` outside of `backend`
+
+```yaml
+
+default:
+  formats:
+    - name: "long-format"
+      value: '{"start_time": "%START_TIME%", "source": "%KUMA_SOURCE_SERVICE%", "destination": "%KUMA_DESTINATION_SERVICE%", "source_address": "%KUMA_SOURCE_ADDRESS_WITHOUT_PORT%", "destination_address": "%UPSTREAM_HOST%", "duration_millis": "%DURATION%", "bytes_received": "%BYTES_RECEIVED%", "bytes_sent": "%BYTES_SENT%"}'
+    - name: "short-format"
+      value: '{"start_time": "%START_TIME%", "source": "%KUMA_SOURCE_SERVICE%", "destination": "%KUMA_DESTINATION_SERVICE%"}'
+  backends:
+    - name: logstash
+      format: "short-format"
+    - name: debug-file
+      format: "long-format"
+```
+
+###### Positive Consequences
+
+* more dry
+
+###### Negative Consequences
+
+* new for the user
+
+#### Type
+
+Stays the in the same place with the `backend`.
+
+### gRPC support
+
+One of the aspects of this new policy is better [support for gGRPC](https://github.com/kumahq/kuma/issues/3324). 
+It seems that nothing [has changed](https://github.com/kumahq/kuma/issues/3324#issuecomment-1008991839) in Envoy and
+there is no new features for gRPC apart from operators that already existed
+(`GRPC_STATUS` - introduced [2 years ago](https://github.com/envoyproxy/envoy/blame/v1.17.0/docs/root/configuration/observability/access_log/usage.rst#L379),
+`GRPC_STATUS_NUMBER` - introduced [4 months ago](https://github.com/envoyproxy/envoy/blame/main/docs/root/configuration/observability/access_log/usage.rst#L598)).
+It might be worth explicitly pointing out these operators in our docs.
+
+## Examples
+
+More full-fledged one using all the options will be described once the desired implementation is chosen.
+
+### Apples and oranges 
+
+```yaml
+type: MeshHTTPRoute
+name: backend-apple-path
+spec:
+  http:
+    rules:
+      - matches:
+          - path:
+              match: PREFIX
+              value: /apples
+```
+```yaml
+type: MeshHTTPRoute
+name: web-orange-path
+spec:
+  http:
+    rules:
+      - matches:
+          - path:
+              match: PREFIX
+              value: /oranges
+```
+```yaml
+type: TrafficLog
+mesh: mesh-1
+name: tl-1
+spec:
+  targetRef:
+    kind: MeshHTTPRoute
+    name: backend-apple-path
+  to:
+    - targetRef:
+        kind: MeshHTTPRoute
+        name: web-orange-path
+      default:
+        backends:
+          - name: logstash
+            format: ""
+          - name: debug-file
```

Comment:
To me this is non-sensical. If we match `backend-apple-path` how can we match something going `to` `web-orange-path`

Reply:
Yeah - you're right - this is the same as https://github.com/kumahq/kuma/pull/4908/files#r955969319 - I'll adjust.